.github/workflows/very_good_core.yaml

```diff
@@ -47,25 +47,25 @@ jobs:
         run: |
           dart pub global activate mason_cli
           mason get
-          mason make very_good_core -c config.json -o output --on-conflict overwrite
+          mason make very_good_core -c config.json --on-conflict overwrite
```

Comment:
Should this not have an `-o very_good_core_output`? As all the other commands after this expect that directory name

Reply:
Ah no, u updated it in the config files. Ignore this comment!

Reply:
it's updated on `very_good_core/config.json`